File: src/main/java/bohdan/markatov/org/dto/comment/CommentResponseDto.java
package bohdan.markatov.org.dto.comment;

import bohdan.markatov.org.dto.user.UserResponseDto;
import java.time.LocalDateTime;
import lombok.Data;

@Data
public class CommentResponseDto {
    private Long id;
    private String text;
    private UserResponseDto author;
    private Long taskId;
    private LocalDateTime publicationDate;
    private boolean isChanged;
}

File: frontend/src/app/services/models/comment-response-dto.ts
/* tslint:disable */
/* eslint-disable */
import {UserResponseDto} from "./user-response-dto";

export interface CommentResponseDto {
  author?: UserResponseDto;
  changed?: boolean;
  id?: number;
  publicationDate?: string;
  taskId?: number;
  text?: string;
}
